src/main/kotlin/no/nav/familie/ef/iverksett/oppgave/OpprettOppgaveForBarnService.kt

```diff
@@ -0,0 +1,69 @@
+package no.nav.familie.ef.iverksett.oppgave
+
+import no.nav.familie.ef.iverksett.iverksetting.IverksettingRepository
+import no.nav.familie.ef.iverksett.iverksetting.domene.Iverksett
+import no.nav.familie.kontrakter.ef.søknad.Fødselsnummer
+import no.nav.familie.kontrakter.felles.oppgave.Oppgavetype
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+import java.time.LocalDate
+
+@Service
+class OpprettOppgaveForBarnService(private val oppgaveClient: OppgaveClient,
+                                   private val iverksettingRepository: IverksettingRepository) {
+
+    private val secureLogger = LoggerFactory.getLogger("secureLogger")
+
+    fun opprettOppgaverForAlleBarnSomFyller(innenAntallUker: Long) {
+        val behandlinger = iverksettingRepository.hentAlleBehandlinger()
+        behandlinger.forEach { iverksett ->
+            iverksett.søker.barn.forEach { barn ->
+                barn.personIdent?.let {
```

Comment:
Tror vi må ta høyde for termindato også, hvis ikke klarer vi aldri å fange opp de som søker før fødsel?

Reply:
Hvordan blir det ? Trenger man termindato når man skal sjekke om noen fyller år en uke frem i tid ? Eller tenker du på noe annet ?

Reply:
Barn har enten personIdent eller termindato ved innvilgelse av en stønad.

Hvis vi skal lage oppgave når dette barnet har fylt 6/12 mnd så må vi se om det er 6/12 mnd siden termindato.

Reply:
Flyttet og tatt høyde for i ef-sak